### PR TITLE
Add Enterprise Docs for team membership & collaboration 

### PR DESCRIPTION
**Pull request by maryelizbeth**
Clarify the difference between access levels on resources in TFE and the types of memberships in organizations. 

[ZenDesk](https://hashicorp.zendesk.com/agent/tickets/4744) 
[Slack](https://hashicorp.slack.com/archives/C2R2ZJZHC/p1505411257000226)

### CODE REVIEW

File: content/source/docs/enterprise/organizations/membership.html.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Owners are the top-most access level within an organization. Owners have complete administrative ability. An owner is able to manage teams and memberships. Owners can also enforce two factor authentication across the organization.
+
+The owners team is a special team made up of all the folks that have `owner` permissons. The team has implied access for all of your resources, but also has the ability to manage your organization.
+
+Owners can also restrict access to resources (Terraform Environments, Packer Builds, and Artifacts). To manage access go to the Access settings for the specific resource and enter the team or collaborators name. At this point you can control the access level for that team: either `read`, `write`, or `admin`.
+
+### Member
+
+Members are simply members of your organization. Owners can organize members into team and restrict their permissions via Access.
```
Comment (nfagerlund) [inline content/source/docs/enterprise/organizations/membership.html.md]: - minus "simply."
- "into team" -> "into teams"
- "...and restrict their permissions. See the \[documentation on access controls\]\(./access.html\) for details."

File: content/source/docs/enterprise/user-accounts/access.html.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+layout: "enterprise"
+page_title: "Access - Accounts - Terraform Enterprise"
+sidebar_current: "docs-enterprise-accounts-access"
+description: |-
+  Terraform Enterprise allows collaboration on resources. However, there are several levels of access.
```
Comment (nfagerlund) [inline content/source/docs/enterprise/user-accounts/access.html.md]: - "...on resources, with several levels..."

File: content/source/docs/enterprise/user-accounts/access.html.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+# Access
+
+Terraform Enterprise allows collaboration on resources. However, there are several access levels available for users.
```
Comment (nfagerlund) [inline content/source/docs/enterprise/user-accounts/access.html.md]: - "...on resources, with several levels..."

File: content/source/docs/enterprise/user-accounts/access.html.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+### Read
+
+Read is the lowest level of access within Terraform Enterprise. Users with read access on a resource can do just that. They are able to view the resource, but are unable to perform any CRUD actions.
```
Comment (nfagerlund) [inline content/source/docs/enterprise/user-accounts/access.html.md]: - "Users with read access on a resource can view it, but are unable to modify or delete it."

**Comment by maryelizbeth** [comment]
@nfagerlund I think I've hit all your edits. You can take another pass when you have a chance!

File: content/source/docs/enterprise/organizations/membership.html.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+### Admin
+
+Admin is the top-most access level within Terraform Enterprise and is reserved for Terraform install admins. Admins can manage users, organizations, and jobs. For example, our support team can clear out stuck plans within Terraform environments or view organization data in order to resolve requests. Admins do not have access to sensitive variables.
+
+### Owner
+
+Owners are the top-most access level within an organization within an organization, and have complete administrative rights. An owner can manage teams and memberships, and can enforce two factor authentication across the organization.
+
+Every organization has a special **owners** team, made up of every user that has `owner` permissions. The team has implied access for all of the organization's resources, but also has the ability to manage the organization.
```
Comment (fprimex) [inline content/source/docs/enterprise/organizations/membership.html.md]: "The team has implied access" -> "The owners team has implied access" would be better IMO as it helps the sentence stand alone in scope if needed.

File: content/source/docs/enterprise/organizations/membership.html.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+Owners can also restrict access to resources (Terraform Environments, Packer Builds, and Artifacts). To manage access go to the Access settings for the specific resource and enter the team or collaborators name, then specify whether they should have `read`, `write`, or `admin`.
```
Comment (fprimex) [inline content/source/docs/enterprise/organizations/membership.html.md]: Dislike the use of the word resources so close to a Terraform context to not mean resources created by Terraform. Are these called resources elsewhere in the docs? If not, and if your list is exhaustive and not examples, then just remove the word and list them. Same for the later use of "resource". Plus minor edits.

> Owners can also restrict access to Terraform Environments, Packer Builds, and Artifacts. To manage access go to the Access settings for the specific environment, build, or artifact, enter the team or collaborators name, and specify whether they should have `read`, `write`, or `admin` access.

File: content/source/docs/enterprise/organizations/membership.html.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+### Member
+
+Members are members of your organization. Owners can organize members into teams and restrict their permissions via Access. See the [documentation on access controls](../user-accounts/access.html) for details.
```
Comment (fprimex) [inline content/source/docs/enterprise/organizations/membership.html.md]: "Members are members" seems repetitive. Maybe:

> Members are Terraform Enterprise users with access to your organization.

File: content/source/docs/enterprise/user-accounts/access.html.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+layout: "enterprise"
+page_title: "Access - Accounts - Terraform Enterprise"
+sidebar_current: "docs-enterprise-accounts-access"
+description: |-
+  Terraform Enterprise allows collaboration on resources, with several levels of access.
+---
+
+# Access
+
+Terraform Enterprise allows collaboration on resources, with several access levels available for users.
```
Comment (fprimex) [inline content/source/docs/enterprise/user-accounts/access.html.md]: I think the comma should be removed.

File: content/source/docs/enterprise/user-accounts/access.html.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+### Read
+
+Read is the lowest level of access within Terraform Enterprise. Users with read access on a resource can view it, but are unable to modify or delete it.
```
Comment (fprimex) [inline content/source/docs/enterprise/user-accounts/access.html.md]: Either remove the comma or add "but they are".

File: content/source/docs/enterprise/user-accounts/access.html.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+
+### Write
+
+Write access allows collaborating users the abilty to view the resource and perform create, update, and edit actions on a resource. Users with write access are not able to destroy resources. For example, users with write access are able to view and edit Terraform environment variables as well as push new configuration.
```
Comment (fprimex) [inline content/source/docs/enterprise/user-accounts/access.html.md]: Suggest "...allows collaborators to view...", but abilty -> ability regardless.

Also, "view and edit Terraform environment" -> "view and edit Terraform Enterprise environment" to keep the product naming/features consistent.

File: content/source/docs/enterprise/user-accounts/access.html.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+
+### Admin
+
+Admin is the top-most user access level within Terraform Enterprise and is reserved for organization owners and select user accounts. Beyond read and write access, admins are given ownership permissions on the resource that allow them to manage collaborations and destroy the resource.
```
Comment (fprimex) [inline content/source/docs/enterprise/user-accounts/access.html.md]: Would again remove "resource" and list the things. "ownership permissions on environments, builds, and artifacts". Unless resource is actually what we call these things, or if we have another approved name for them.

File: content/source/docs/enterprise/organizations/membership.html.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+  Terraform Enterprise allows collaboration in organizations. However, there are several membership levels within organizations.
+---
+
+# Membership
+
+Terraform Enterprise allows collaboration in organizations. However, there are several membership levels within organizations.
+
+### Admin
+
+Admin is the top-most access level within Terraform Enterprise and is reserved for Terraform install admins. Admins can manage users, organizations, and jobs. For example, our support team can clear out stuck plans within Terraform environments or view organization data in order to resolve requests. Admins do not have access to sensitive variables.
```
[the remark this answers is not in the document]
Comment (maryelizbeth) [reply content/source/docs/enterprise/organizations/membership.html.md]: @fprimex this was a request from @skierkowski to make clear that we are unable to read sensitive vars when viewing customer data. it was also specifically called out in the req from the customer as well.

File: content/source/docs/enterprise/organizations/membership.html.md (hunk 6 above)
Comment (maryelizbeth) [reply content/source/docs/enterprise/organizations/membership.html.md]: we refer to them as resources in the atlas UI as well.

File: content/source/docs/enterprise/user-accounts/access.html.md (hunk 11 above)
Comment (maryelizbeth) [reply content/source/docs/enterprise/user-accounts/access.html.md]: they're called "resources" in the UI, but I don't mind listing them here as well.

**Pull request merged**